src/common/Aside.tsx

```diff
@@ -5,7 +5,7 @@ interface AsideProps {
 }
 
 const Aside: React.FunctionComponent<AsideProps> = ({ children }) => {
-  return <aside className="col col--3-lg col--4-md">{children}</aside>;
+  return <aside className="col col--2-lg col--1-md">{children}</aside>;
```

Comment:
See comment on `index.css`

Reply:
Same as the above css comment. Without the css it's not displaying the text correctly. However, you said it's having negative affects on other parts of the code using the css? I will take a look and see if I can get it to display, but it will be tricky if no css is used.

Reply:
I removed the aboved CSS (changed "card-demo-pb-1" back to "card-demo pb-1" and "col--2-lg col--1-md" back to "col--3 col--4-md" and the output looks as below. It places the Article Upload textbox on a separate line. Is this ok?
![image](https://user-images.githubusercontent.com/52865683/145738464-4bb8b4ee-5c8a-4c12-ae2d-3fb83d1177ee.png)